src/pages/home/sidebar/OptionRow.js

```diff
@@ -130,12 +130,12 @@ const OptionRow = ({
                                         avatarImageURLs={option.icons}
                                         optionIsFocused={optionIsFocused}
                                         size={mode === 'compact' ? 'small' : 'default'}
-                                        styles={hovered && !optionIsFocused && {
+                                        styles={(hovered && !optionIsFocused) ? {
                                             secondAvatar: {
                                                 backgroundColor: themeColors.sidebarHover,
                                                 borderColor: themeColors.sidebarHover,
                                             },
-                                        }}
+                                        } : undefined}
```

Comment:
Just wondering, what was the warning here? Having `undefined` here seems less clean

Reply:
also asked Rajat to fix this up in another PR